### ISSUE TITLE
Fix render issue. if use the MultiShaderBufferedDrawNode

### ISSUE THREAD

**Issue by andy840119** [karaoke-dev/osu-framework-font#159]
![image](https://user-images.githubusercontent.com/9100368/165489723-7015ddd9-2d9b-45c2-bffe-13f6a2a1ee12.png)
if change the `ComputeScreenSpaceDrawQuad` in the `LyricSpriteText`
```diff
-            var newRectangle = DrawRectangle.Scale(2);
+            var newRectangle = DrawRectangle.Scale(1.2f);
```

**Comment by andy840119** [comment]
I guess it will have the same issue if set the scale to 2.
But not that obviously.

**Comment by andy840119** [comment]
Should test with step shader with no outline shader.

**Comment by andy840119** [comment]
![image](https://user-images.githubusercontent.com/9100368/166151173-9b6d9a08-30e1-47ac-af59-f3a65c32846a.png)
Damn, that weird shader effect is out of the lyric sprite text itself.


![image](https://user-images.githubusercontent.com/9100368/166151278-302c5487-2266-4b0f-8207-4e0154e9d805.png)
I guess karaoke sprite text is a sus.

Also, should deal with `ComputeScreenSpaceDrawQuad()` in the `KaraokeSpriteText`

**Comment by andy840119** [comment]
![image](https://user-images.githubusercontent.com/9100368/166151845-a4f5adbb-5413-4f5d-a0fb-2441be0953ec.png)
Look at the border, it's sus.
Maybe should test masking container with:
- two frame buffer container.
- `TestShaderContainer` with masking container?
- osu sprite text with masking container?

**Comment by andy840119** [comment]
Note:
Maybe write `SingleShaderBufferedDrawNode` first for checking if step shader will cause the same issue.
Because `MultiShaderBufferedDrawNode` has recycle issue, and it might not be used in the V1 karaoke system.

**Comment by andy840119** [comment]
https://github.com/karaoke-dev/osu-framework-font/blob/master/osu.Framework.Font/Graphics/MultiShaderBufferedDrawNode.cs#L115
Guess i got the reason:
Should not use source and target as same frame buffer.

**Comment by andy840119** [comment]
Guess it will not happened after #169 merged.
But this issue will be closed until fixed in the `MultiShaderBufferedDrawNode`.

**Comment by andy840119** [comment]
Trying to solve in the #247
So this issue can be close.

**Comment by andy840119** [comment]
Wait.
Re-open it because this issue is focus on render broken.

**Comment by andy840119** [comment]
Guess implemented in the #331.
